lua/quickmath/init.lua

```diff
@@ -116,6 +120,227 @@ function go_eol()
 
 end
 
+function vec(...)
```

Comment:
Just a small comment, the syntax for vector initialization with `a = vec {1,2,3}` instead of `a = vec(1,2,3)` could be nicer. I'll let you decide because it's strictly design and it's "your" code.